.readthedocs.yaml

```diff
@@ -21,6 +26,7 @@ python:
   install:
     - requirements: requirements-doc.txt
     - requirements: requirements.txt
+    - path: . # path to the package relative to the root
```

Comment:
Just curious. Why is specifying `path` necessary?

Reply:
What ended up happening was the build was failing. I looked at hdmf to see how it was set up there. Looking at the error, it was not properly installing the requirements when building the docs. I looked at the documentation for how to setup this yaml and it says the "path" would be the path to the requirements file. I confirmed that in hdmf it was also present. These changes allowed for the docs to properly find the requirements and be built.